intezeranalyze_connector.py

```diff
@@ -33,6 +35,8 @@ def __init__(self):
         # modify this as you deem fit.
         self._base_url = None
```

Comment:
make sure the base url contains /api like https://analyze.intezer.com/api/ or add it to the rest of the routes

Reply:
it is contain /api, it's located on phintezeranalyze.json